tests/test_processing.py

```diff
@@ -60,63 +64,139 @@ def test_find_peaks_empty(self):
 
     def test_gqrs(self):
 
-        record = wfdb.rdrecord('sample-data/100', channels=[0],
-                                  sampfrom=9998, sampto=19998, physical=False)
-
-        expected_peaks = [271, 580, 884, 1181, 1469, 1770, 2055, 2339, 2634,
-                          2939, 3255, 3551, 3831, 4120, 4412, 4700, 5000, 5299,
-                          5596, 5889, 6172, 6454, 6744, 7047, 7347, 7646, 7936,
-                          8216, 8503, 8785, 9070, 9377, 9682]
-
-        peaks = processing.gqrs_detect(d_sig=record.d_signal[:,0],
-                                       fs=record.fs,
-                                       adc_gain=record.adc_gain[0],
-                                       adc_zero=record.adc_zero[0],
-                                       threshold=1.0)
+        record = wfdb.rdrecord(
+            "sample-data/100",
+            channels=[0],
+            sampfrom=9998,
+            sampto=19998,
+            physical=False,
+        )
+
+        expected_peaks = [
+            271,
+            580,
+            884,
+            1181,
+            1469,
+            1770,
+            2055,
+            2339,
+            2634,
+            2939,
+            3255,
+            3551,
+            3831,
```

Comment:
Is this the PEP8 way to do this? What's wrong with the other one if it's under 80 characters?